docs/guide/protocols/septop.rst

```diff
@@ -0,0 +1,131 @@
+Separated Topologies Protocol
+=============================
+
+Overview
+--------
+
+The :class:`SepTopProtocol <.SepTopProtocol>` [1]_, [2]_ calculates the difference in binding free energy between two ligands.
+This protocol essentially performs two absolute binding free energy calculations simultaneously in opposite directions,
+by (alchemically) inserting one ligand into the binding site, while removing the other ligand at the same time.
+In contrast to the :ref:`RelativeHybridTopologyProtocol <userguide_relative_hybrid_topology_protocol>`, the two ligand topologies are
+completely separate (meaning there is no common core), making atom mapping unnecessary and allowing transformations between chemically diverse ligands.
+
+The relative binding free energy is calculated through a thermodynamic cycle by transforming one ligand into the other ligand
+both in the solvent and in the binding site.
+
+Restraints are required to keep the weakly
+coupled and fully decoupled ligand in the binding site region and thereby reduce the phase
+space that needs to be sampled. In the :class:`SepTopProtocol <.SepTopProtocol>`
+we apply orientational, or Boresch-style, restraints, as described below.
+
+In this cycle, the interactions of one molecule are turned off while simultaneously turning on interactions of the other molecule both in the solvent and complex phases.
+The relative binding free energy is then obtained via summation of free energy differences along the thermodynamic cycle.
+
+.. figure:: img/septop_cycle.png
+   :scale: 50%
+
+   Thermodynamic cycle for the SepTop free energy protocol.
+
+Scientific Details
+------------------
+
+Orientational restraints
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+Orientational, or Boresch-style, restraints are automaticallly (unless manually specified) applied between three protein and three ligand atoms using one bond,
+two angle, and three dihedral restraints. Reference atoms are picked based on different criteria, such as the root mean squared
+fluctuation of the atoms in a short MD simulation, the secondary structure of the protein, and the distance between atoms, based on heuristics from Baumann et al. [2]_.
```

Comment:
TODO (issued raised): update the docs here (and in ABFEs) to reflect the new picking, i.e. it's getting closer to a mix between Hannah's appproach and MDRestraintsGenerator.